crates/sui-types/src/crypto.rs

```diff
@@ -1636,8 +1653,9 @@ pub enum SignatureScheme {
     ED25519,
     Secp256k1,
     Secp256r1,
-    BLS12381, // This is currently not supported for user Sui Address.
     MultiSig,
+    BLS12381, // This is currently not supported for user Sui Address.
```

Comment:
we cannot reorder previously existing variants, this will break deserialization of previously committed transactions

Reply:
or wait, does this do custom serialization using the flag bytes?

Reply:
the flag byte is defined below unrelated to the enum serde itself. revised to not change order